docs/SUMMARY.md

```diff
@@ -39,7 +39,16 @@
     * [Excluding patterns](earthfile/earthignore.md)
 * [The `earthly` command](earthly-command/earthly-command.md)
 * [Configuration reference](earthly-config/earthly-config.md)
-* [CI integration](./ci-integration.md)
+* [CI integration](ci-integration/ci-integration.md)
+  * [Building An Image](ci-integration/building-an-image.md)
```

Comment:
`Building An Image` could be mistaken for something else in the context of a build system. Should we call this `The Earthly Image` instead?

Reply:
Yeah I wrestled with this. `The Earthly Image` makes it sound like we are discussing our image... one alternate I has used in the past was "Adding Earthly to a CI Image"?

Reply:
That works